packages/core/database/lib/entity-manager/index.js

```diff
@@ -1164,6 +1170,53 @@ const createEntityManager = (db) => {
       }
     },
 
+    // TODO: Clone polymorphic relations
+    // TODO: Excluded relation attributes
+    /**
+     *
+     * @param {*} uid - uid of the entity to clone
```

Comment:
Should these param docs be of type: uid: string, id: number, cloneId: number?

Reply:
yep, good catch! :)